hudi-client/hudi-client-common/src/test/java/org/apache/hudi/table/action/TestCleanPlanner.java

```diff
@@ -74,7 +76,8 @@
 import static org.mockito.Mockito.when;
 
 public class TestCleanPlanner {
-  private static final Configuration CONF = new Configuration();
+  private static final StorageConfiguration<Configuration> CONF =
+      HadoopFSUtils.getStorageConf(new Configuration());
```

Comment:
Can we use `HoodieTestUtils.getDefaultStorageConf()` ?

Reply:
Fixed.